### DISCUSSION TITLE
Respect transparent backgrounds

### DISCUSSION THREAD

**Discussion by pauubach** [advanced-cropper/vue-advanced-cropper#266] [Ideas]
It would be great if images with transparent parts have them transparent when cropped.

**Comment by Norserium** [comment]
@pauubach, actually, the cropper respects transparency. Try that in [this sandbox](https://codesandbox.io/p/sandbox/vue-advanced-cropper-options-api-yrxou?file=%2Fsrc%2FApp.vue) with this image:
![cats](https://github.com/advanced-cropper/vue-advanced-cropper/assets/9862719/0a6c3b73-b687-4024-97fd-8b974a8c8de3)
